client/src/components/DataFiles/DataFilesModals/DataFilesManageProjectModal.jsx

```diff
@@ -116,7 +120,7 @@ const DataFilesManageProjectModal = () => {
         className="dataFilesModal"
       >
         <ModalHeader toggle={toggle} charCode="&#xe912;">
-          Manage Team
+          {isUserOrGuest ? 'View' : 'Manage'} Team
```

Comment:
I see the issue that we only have `owner` and `edit` roles.

@edmondsgarrett I'm thinking we should amend these conditionals to check for a `read` role, which we can either add here or in a follow on task if it's too complicated. What do you think?

Reply:
Yeah, this is what I was originally thinking, I just wasn't sure where I could add this in. I'll take a look at this again.

Reply:
From looking through our redux and django stuff on project metadata, it seems to me like the fix would be to change the PI/Co-PI/Team members approach in the metadata model to PI/Co-PI/Users/Guests, which we could then use to add a `read` role in redux. The trouble I see with this is that the `team-members` portion of the role metadata touches a lot of other stuff in the back end, so this update would likely be big. What do you think? I haven't been able to think of a more streamlined solution up to this point.